qfieldsync/core/cloud_api.py

```diff
@@ -610,20 +610,6 @@ def projects(self) -> Optional[List[CloudProject]]:
     def error_reason(self) -> str:
         return self._error_reason
 
-    @property
-    def is_currently_open_project_cloud_local(self) -> bool:
```

Comment:
@suricactus , it is _not_ the same as currently_open_project. The latter *requires* a connected state and a valid project cache, whereas this function only checks whether the local directory is registered as a mirror of a server cloud project.

Without this, the qfieldsync toolbar's cloud synchronization button can't be enabled / disabled properly when not connected, which isn't great UX.

Reply:
Ah, I see now. You are right. A comment can help next time to highlight the difference.

Reply:
What I see as a problem is there might be a `project_id` configuration in the local paths that no longer exists.

Reply:
Added docs and a check for the project_id, if projects are present